Makefile

```diff
@@ -2,15 +2,14 @@ SHELL := bash -euo pipefail
 
 .PHONY: data/sfs-redcap.sqlite venv requirements.txt
 
-data/sfs-redcap.sqlite: data/record-barcodes.ndjson indexes.sql derived-tables.sql
+data/sfs-redcap.sqlite: data/record-barcodes.csv import.sql indexes.sql derived-tables.sql
 	sqlite3 $@ 'PRAGMA journal_mode=WAL;'
 	chmod -v g+w $@
-	sqlite-utils insert --nl --truncate $@ record_barcodes_new $<
-	sqlite3 $@ 'begin transaction; drop table if exists record_barcodes; alter table record_barcodes_new rename to record_barcodes; commit;'
+	sqlite3 $@ < import.sql
```

Comment:
If you wanted to avoid a sql file you might be able to  use "-cmd" like this:
sqlite3 $@ -cmd '.mode csv' '.import data/record-barcodes.csv record_barcodes_new'

But I haven't tried "-cmd" with a regular SQL command like 
drop table if exists record_barcodes;

Reply:
Nod. One reason to keep it in the Makefile might be if we wanted to interpolate the input file name instead of hardcoding it in the SQL script.

But I'm fine for now with putting less into the Makefile and keeping it in a file instead, like the other SQL scripts. Do you think it'd be better to do it differently?

Reply:
Both options are good. I liked having one fewer file to deal with, but there's no compelling reason to put it into the Makefile.